DEVELOPERS_GUIDE.md

```diff
@@ -1,48 +1,48 @@
+Make sure that KaiZen-OpenAPI-Editor plugins are included to the configuration, for example, you can select the "All workspace and enabled target plag-ins" in the "Plug-ins" tab:
 
 ## 3. Launch: using a local update site built from sources
 
-From inside your SwagEdit folder, run the following command:
+From inside your KaiZen-OpenAPI-Editor folder, run the following command:
 
 ```
 mvn clean verify
 ```
 
-This command will build the project and generate an update site under the folder `SwagEdit/com.reprezen.swagedit.repository/target/repository`.
+This command will build the project and generate an update site under the folder `KaiZen-OpenAPI-Editor/com.reprezen.swagedit.repository/target/repository`.
 
-You can now install SwagEdit into your Eclipse by clicking on `Help > Install New Software... > Add...`
-This will show a dialog box from where you can select the location of the update site.
-Click on `Local...` and select the folder `SwagEdit/com.reprezen.swagedit.repository/target/repository` and then click `Ok`.
+You can now install KaiZen OpenAPI Editor into your Eclipse by clicking on `Help > Install New Software... > Add...`
+This will show a dialog box where you can select the location of the update site.
+Click on `Local...` and select the folder `KaiZen-OpenAPI-Editor/com.reprezen.swagedit.repository/target/repository` and then click `Ok`.
 
-You can now select and install SwagEdit from the Eclipse update manager.
+You can now select and install KaiZen-OpenAPI-Editor from the Eclipse update manager.
 
 # Style Guide
-Please import [RepreZen Java Code Formatter](https://raw.githubusercontent.com/RepreZen/SwagEdit/master/etc/dev-env/ModSquad_formatter_profile.xml) to your Eclipse IDE.
+Please import [RepreZen Java Code Formatter](https://raw.githubusercontent.com/RepreZen/KaiZen-OpenAPI-Editor/master/etc/dev-env/ModSquad_formatter_profile.xml) to your Eclipse IDE.
 
-# SwagEdit Architecture
-![SwagEdit Architecture](https://cloud.githubusercontent.com/assets/644582/13757221/cf31b4e8-e9f9-11e5-8e6b-8aeb26fc3ac9.png)
+# Architecture
+![KaiZen OpenAPI Editor Architecture](https://cloud.githubusercontent.com/assets/644582/13757221/cf31b4e8-e9f9-11e5-8e6b-8aeb26fc3ac9.png)
 
 # JSON Schema
-Validation, Code-assist, Outline, and Quick-Outline are built based on Swagger JSON Schema which is analized by our model, you can find more details in [#198 SwagEdit Model](https://github.com/RepreZen/SwagEdit/issues/198).
+Validation, code assist, outline, and quick outline are built based on JSON Schema which is analyzed by our model. You can find more details in [#198 KaiZen OpenAPI Editor Model](https://github.com/RepreZen/KaiZen-OpenAPI-Editor/issues/198).
 ![JSON Schema Model](http://i.imgur.com/h38zU2C.png)
 ### Validation
 See `com.reprezen.swagedit.validation.Validator.isSchemaDefinition(AbstractNode)`
```

Comment:
This class was moved to the new `core` plugin, its new FQN is `com.reprezen.swagedit.core.validation.Validator`